### PR TITLE
Jenkins envvars to continue the trace

### PR DESCRIPTION
**Pull request by drodriguezhdez**

### CODE REVIEW

File: docs/pipelines/jenkins.md [hunk 1]
```diff
@@ -96,6 +96,24 @@ pipeline {
 }
 {% endhighlight %}
 
+## Continue the trace with your own spans
+
+### Using the datadog-ci CLI
+
+TBD
```
Comment (drodriguezhdez) [inline docs/pipelines/jenkins.md:103]: @juan-fernandez maybe you can add the instructions (or a link to the instructions) here?

Comment (juan-fernandez) [reply docs/pipelines/jenkins.md:103]: I'll push the change once https://github.com/DataDog/ciapp-alpha-docs/pull/24 is merged

Comment (drodriguezhdez) [reply docs/pipelines/jenkins.md:103]: @juan-fernandez do you have already the change prepared, or should I add it?

Comment (juan-fernandez) [reply docs/pipelines/jenkins.md:103]: I can do it in a bit or you can take it. I have nothing yet

Comment (drodriguezhdez) [reply docs/pipelines/jenkins.md]: No worries. I've just added the change 👍🏼

File: docs/pipelines/jenkins.md [hunk 2]
```diff
@@ -121,6 +121,28 @@ pipeline {
...
 
+## Continue the trace with your own spans
+
+### Using the datadog-ci CLI
+
+You can wrap your shell commands using the `datadog-ci` CLI. 
+
+This will automatically continue your trace creating a new span with every command.
+
+You can check the [datadog-ci CLI](/custom-instrumentation/datadog-ci) documentation to know how to install and using it.
```
Comment (juan-fernandez) [inline docs/pipelines/jenkins.md:132]:
```suggestion
You can check the [datadog-ci CLI](/custom-instrumentation/datadog-ci) documentation to know how to install and use it.
```

**Review by juan-fernandez — APPROVED** [review]

**Pull request merged**